docs/contributing/hacking.rst

```diff
@@ -139,10 +313,38 @@ Make sure it meets the following requirements:
     ``172.16.0.0/12``, or ``192.168.0.0/16``, you'll have to modify ``contrib/coreos/user-data.example``
     and whitelist your development registry so the daemons can pull your custom components.
 
+Initial Platform Build
+----------------------
+
+The full environment is prepared. You can now build Deis from source code and
+then run the platform.
+
+We'll do three steps together:
+
+- Build the source
+- Update our local cluster with a dev release
+- Start the platform
+
+.. code-block:: console
+
+    $ make build    # Build the entire codebase. This will take a while.
+    $ make dev-release  # Create a local dev release.
+    $ deisctl start platform
```

Comment:
`make deploy` works on a fresh cluster (I use it all the time). It just stops units before starting them, which is pretty much instant because they haven't been started.

I'm just a fan of cutting down the number of typed commands because I am lazy and I don't have to pay attention to the deploy process to start the next task.